.github/workflows/ci.yaml

```diff
@@ -178,17 +181,60 @@ jobs:
         run: make clean
         working-directory: test
 
-  ramenctl-test:
-    name: ramenctl tests
-    runs-on: ubuntu-22.04
+  drenv-macos:
+    name: drenv
+    strategy:
+      matrix:
+        os:
+          - macos-14
```

Comment:
This is a VM on M1 machine, so it does not support nested virtualization. We can try to use macos-13 (x86 vm). It it support nested virtualization we can run lima vm and run all tests.

Reply:
We can use `macos-13` which is intel machine and it support nested virtualization:
https://docs.github.com/en/actions/using-github-hosted-runners/using-github-hosted-runners/about-github-hosted-runners#viewing-available-runners-for-a-repository

lima uses this to run vms on github:
https://github.com/lima-vm/lima/blob/master/.github/workflows/test.yml#L404